WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsTimezoneDialog.java

```diff
@@ -185,6 +185,8 @@ private void loadTimezones(@NonNull String responseJson) {
             // sort by label
             Collections.sort(timezones, new Comparator<Timezone>() {
                 public int compare(Timezone t1, Timezone t2) {
+                    // manually perform the StringUtils.compare() function. Some devices seem to be missing it.
```

Comment:
Looks like we can drop this comment now.

Reply:
Done with b1b45dc!